core/templates/dev/head/base_components/WarningLoaderDirective.ts

```diff
@@ -30,8 +31,11 @@ oppia.directive('warningLoader', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/base_components/warning_loader_directive.html'),
       controllerAs: '$ctrl',
-      controller: [
-        function() {}
+      controller: ['AlertsService',
+        function(AlertsService) {
+          var ctrl = this;
+          ctrl.AlertsService = AlertsService;
```

Comment:
Can we do `ctrl.warnings = AlertsService.warnings;`?

Reply:
I think that won't work, we will have to bind AlertService.

Reply:
Okay